sklearn/linear_model/tests/test_ridge.py

```diff
@@ -527,6 +529,29 @@ def test_ridgecv_store_cv_values():
     assert_equal(r.cv_values_.shape, (n_samples, n_responses, n_alphas))
 
 
+def test_ridgecv_sample_weight():
+    rng = np.random.RandomState(0)
+    alphas = (0.1, 1.0, 10.0)
+
+    for n_samples, n_features in ((6, 5), (5, 10)):
```

Comment:
> I agree that one is enough. Could you see my comment below? I want to
> understand why test_ridge_sample_weights uses two cases for n_samples and
> n_features. Thanks!

Ha, that's a good point: you need to test those 2 strategies. Can you add
a comment about that.

The 2 strategies are because the 2 options are faster in each case,
n_samples > n_features or the opposite.